conda/environments/all_cuda-115_arch-x86_64.yaml

```diff
@@ -0,0 +1,27 @@
+# This file is generated by `rapids-dependency-file-generator`.
+# To make changes, edit dependencies.yaml and run `rapids-dependency-file-generator`.
+channels:
+- rapidsai
+- rapidsai-nightly
+- conda-forge
+- nvidia
+dependencies:
+- clang-tools=11.1.0
+- clang=11.1.0
+- cmake>=3.23.1
+- cudatoolkit=11.5
+- cudf=22.12.*
+- cython>=0.29,<0.30
+- gdal>=3.0.2
```

Comment:
Don't we need 3.5?

Reply:
Updated to reflect https://github.com/rapidsai/cuspatial/pull/806